internal/cli/atlas_alert_config_create_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mcli/internal/fixtures"
+	"github.com/mongodb/mcli/internal/mocks"
+)
+
+func TestAtlasAlertConfigCreate_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockAlertConfigurationCreator(ctrl)
+
+	defer ctrl.Finish()
+
+	t.Run("flags run", func(t *testing.T) {
```

Comment:
Since this won't run from a file I think is safe to move the test out of this closure